gradio/components.py

```diff
@@ -3269,10 +3269,6 @@ def postprocess(
                     running_category = category
                 elif category == running_category:
                     running_text += self.adjacent_separator + text
-                elif not text:
```

Comment:
This got added by mistake no?

Reply:
Yeah it looks like it. That's why the test is failing!